regolith/helpers/u_milestonehelper.py

```diff
@@ -13,28 +13,29 @@
 ALLOWED_STATI = {"p":"proposed", "s":"started", "f":"finished", "b":"back_burner","c":"converged"}
 
 def subparser(subpi):
-    subpi.add_argument("projectum_id", help="the id of the projectum")
+    subpi.add_argument("projectum_id", help="The id of the projectum.")
     subpi.add_argument("-v", "--verbose", action="store_true",
-                        help="gives a list of the milestones available to update.")
+                        help="Gives a list of the milestones available to update "
```

Comment:
"Increases the verbosity of the output."

Reply:
done!